package.json

```diff
@@ -69,7 +69,7 @@
     "@istanbuljs/nyc-config-typescript": "^1.0.2",
     "@types/chai": "^4.3.1",
     "@types/mocha": "^9.1.1",
-    "@types/node": "^14.18.22",
+    "@types/node": "^14.18.23",
```

Comment:
I see that there are merge conflicts. If you back out this update and the `package-lock.json` updates, the conflicts should go away. We update dependencies regularly, so you don't need to worry about that.

Reply:
Okay I changed, it's because I had an typescript error with `Buffer` type

Reply:
So, with this change, are you now having problems with the typing? If so, we can make a PR to update those types for you.

Reply:
Hum the type looks fine, I think my VScode was buggy and didn't detect the `Buffer` type